src/tests/ftest/util/server_utils_params.py

```diff
@@ -433,16 +434,17 @@ class EngineYamlParameters(YamlParameters):
     # Engine environment variables that are required by provider type.
     REQUIRED_ENV_VARS = {
         "common": [
-            "D_LOG_FILE_APPEND_PID=1",
-            "CRT_EVENT_DELAY=1",
-            "COVFILE=/tmp/test.cov"],
+            EnvironmentalVariable("D_LOG_FILE_APPEND_PID", "1"),
+            EnvironmentalVariable("DAOS_POOL_RF", "4", True),
```

Comment:
Why is this necessary? Couldn't we just let all of these be overridden and then we don't need the new class `EnvironmentalVariable` at all?